opm/simulators/flow/LogOutputHelper.cpp

```diff
@@ -306,34 +308,77 @@ fip(const Inplace& inplace,
     }
 }
 
+
 template<class Scalar>
 void LogOutputHelper<Scalar>::
-fipResv(const Inplace& inplace) const
+fipResv(const Inplace& inplace, const std::string& name) const
 {
     {
         std::unordered_map<Inplace::Phase, Scalar> current_values;
 
-        for (const auto& phase : Inplace::phases()) {
+        for (const auto& phase : Inplace::phases()) 
             current_values[phase] = inplace.get(phase);
-        }
+            
+        Scalar field_dyn_pv = 0.0;
+        
+        for (std::size_t reg = 1; reg <= inplace.max_region(name); ++reg)
```

Comment:
[`Inplace::max_region()`](https://github.com/OPM/opm-common/blob/301759536b1bdf935c56891df0e7d045028699e2/src/opm/output/eclipse/Inplace.cpp#L115) is somewhat expensive, so maybe we could write this as
```C++
for (auto nreg = inplace.max_region(name), reg = 0*nreg + 1; reg <= nreg; ++reg) {
    // ...
}
```
instead, just to ensure that we don't compute `max_region()` more than once?